package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@segment/analytics.js-integration-doubleclick-floodlight",
+  "description": "The DoubleClick Floodlight analytics.js integration.",
+  "version": "1.0.0",
+  "keywords": [
+    "analytics.js",
+    "analytics.js-integration",
+    "segment",
+    "doubleclick-floodlight",
+    "doubleclick",
+    "floodlight"
+  ],
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "make test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/segment-integrations/analytics.js-integration-doubleclick-floodlight.git"
+  },
+  "author": "Segment <friends@segment.com>",
+  "license": "SEE LICENSE IN LICENSE",
+  "bugs": {
+    "url": "https://github.com/segment-integrations/analytics.js-integration-doubleclick-floodlight/issues"
+  },
+  "homepage": "https://github.com/segment-integrations/analytics.js-integration-doubleclick-floodlight#readme",
+  "dependencies": {
+    "@ndhoule/each": "^2.0.1",
+    "@ndhoule/foldl": "^2.0.1",
+    "@segment/analytics.js-integration": "^3.2.0",
+    "qs": "^6.3.0"
```

Comment:
The latest version of `qs` is 11kb minified 😱 . Could we use `"component-querystring": "^2.0.0"` instead so we dont add any new dependencies?